apps/opencs/model/world/refidadapterimp.cpp

```diff
@@ -653,6 +672,21 @@ void CSMWorld::LightRefIdAdapter::setData (const RefIdColumn *column, RefIdData&
         light.mData.mColor = value.toInt();
     else if (column==mColumns.mSound)
         light.mSound = value.toString().toUtf8().constData();
+    else if (column == mColumns.mEmitterType)
+    {
+        int mask = ~(ESM::Light::Flicker | ESM::Light::FlickerSlow | ESM::Light::Pulse | ESM::Light::PulseSlow);
+
+        if (value.toInt() == 0)
```

Comment:
For future reference, this is a candidate for a switch-case or even a simple array. But never mind for now.